dotnet-interactive/build-and-install-dotnet-interactive.ps1

```diff
@@ -3,11 +3,9 @@ $ErrorActionPreference = "Stop"
 
 $thisDir = Split-Path -Parent $PSCommandPath
 $toolLocation = ""
-$toolVersion = ""
 if (Test-Path 'env:DisableArcade') {
-    dotnet pack "$thisDir\dotnet-interactive.csproj" /p:Version=0.0.0
+    dotnet pack "$thisDir\dotnet-interactive.csproj"
```

Comment:
Why do away with the version number?

Installing the specific version ensures that we don't grab a version from a nuget feed if it's higher than the local build, or if the local build is missing.